src/Commands/UpdateTenant.php

```diff
@@ -73,18 +74,26 @@ public function handle()
             'idp_x509_cert' => $this->option('x509cert'),
             'relay_state_url' => $this->option('relayStateUrl'),
             'name_id_format' => $this->resolveNameIdFormat(),
-            'metadata' => ConsoleHelper::stringToArray($this->option('metadata'))
+            'metadata' => ConsoleHelper::stringToArray($this->option('metadata')),
         ]));
 
-        if(!$tenant->save()) {
+        // Update separately than the above array_filter, so caller can actually set to empty value.
+        // Note: ->option() value is NULL if _not passed at all_, and empty string if passed
+        // with no value, e.g. --spEntityIdOverride=
+        $spEntityIdOptionWasPassed = !is_null($this->option('spEntityIdOverride'));
```

Comment:
Personal preference, I think its more readable and more explicit.

```suggestion
        $spEntityIdOptionWasPassed = $this->option('spEntityIdOverride') !== null;
```

Reply:
ok 🤷